File: Lesson/src/lesson2/forEach/SubMenuBuilder.java
package lesson2.forEach;

import MenuBuilder.ConsoleMenuBuilder;
import lesson2.forEach.actions.*;

public class SubMenuBuilder {
    static public ConsoleMenuBuilder getMenu(String prnTitle) {
        return new ConsoleMenuBuilder(prnTitle)
                .AddMenuItem("1", "Colors array", new OutColors())
                .AddMenuItem("2", "for-each for string", new OutStringByChar())
                .AddMenuItem("0", "Back to previous menu", null, true);
    }
}

File: Lesson/src/lesson12/SubMenuBuilder.kt
package lesson12

import MenuBuilder.ConsoleMenuBuilder
import lesson12.actions.Lesson12


class SubMenuBuilder {
    companion object {
        @JvmStatic
        fun getMenu(prnTitle: String): ConsoleMenuBuilder {
            val LESSON12 = "Lesson12"

            return ConsoleMenuBuilder(prnTitle)
                    .AddMenuItem("1", LESSON12, Lesson12())
                    .AddMenuItem("0", "Go to previous menu", null, true)
        }
    }
}
